src/System.CommandLine.Rendering/ConsoleExtensions.cs

```diff
@@ -26,5 +26,31 @@ public static OutputMode DetectOutputMode(this IConsole console)
                 return OutputMode.File;
             }
         }
+
+        public static ITerminal TryGetTerminal(
```

Comment:
Given that this doesn't follow the normal Try* signature pattern, I would suggest simply calling it `GetTerminal`